Hearthstone Deck Tracker/Windows/OverlayWindow.Tooltips.cs

```diff
@@ -69,33 +29,33 @@ private void UpdateCardTooltip()
 			var relativeCardMark = _cardMarks.Select(x => new { Label = x, Pos = x.PointFromScreen(new Point(pos.X, pos.Y)) });
 			var visibility = (Config.Instance.OverlayCardToolTips && !Config.Instance.OverlaySecretToolTipsOnly)
 								 ? Visible : Hidden;
-			ToolTipCard.CardSetToolTip.Visibility = Config.Instance.OverlaySetToolTips ? Visible : Collapsed;
-			ToolTipCard.CreatedByVisibility = Collapsed;
 
 			var cardMark =
 				relativeCardMark.FirstOrDefault(
 												x =>
 												x.Label.IsVisible && PointInsideControl(x.Pos, x.Label.ActualWidth, x.Label.ActualHeight, new Thickness(3, 1, 7, 1)));
+			
+			ToolTipCardBlock.CreatedByVisibility = Collapsed;
 			if(!Config.Instance.HideOpponentCardMarks && cardMark != null)
 			{
 				var index = _cardMarks.IndexOf(cardMark.Label);
 				var card = _game.Opponent.Hand.FirstOrDefault(x => x.GetTag(GameTag.ZONE_POSITION) == index + 1 && x.HasCardId && !x.Info.Hidden)?.Card;
 				var creatorCard = _cardMarks[index].SourceCard;
-				if(card != null || creatorCard != null)
+				if(card == null && creatorCard != null)
 				{
```

Comment:
I think this check needs to stay the way it was? There is two "types" of tooltips we want to show here. 1) For cards where we know the creator we want to show the creator card and the "Created by {creator}" label. 2) For cards where we know the cardId, e.g. because the card was returned to hand or we were able to predict it, we want to show that card but *hide* the created by label.